_includes/overall-schedule.md

```diff
@@ -25,3 +25,90 @@ Organizers will inform participants of the week schedule by email.
 {%- assign w-desc = "" %}
 {%- endfor %}
 {%- endfor %}
+
+<script type="application/ld+json" >
+{
+  "@context": "https://schema.org",
+  "@id": "https://4000-docelijah-openlifescien-imqv6d2ud7l.ws-eu71.gitpod.io/ols-6/schedule/",
+  "@type": "Course",
+  "dct:conformsTo": "https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08",
+  "description": "The OLS program is for people interested in applying open principles in their work and becoming Open Science ambassadors in their communities.",
```

Comment:
You can use the description in the `_config.yml` file using `{{ site.description }}` instead of re-writing it

Reply:
Thanks @bebatut